src/__tests__/lib/utils/hooks/useGetInfoFromIds.test.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import useGetInfoFromIds, {
   UseGetInfoFromIdsProps,
 } from '../../../../lib/utils/hooks/useGetInfoFromIds'
-import { act } from 'react-dom/test-utils'
+import { act } from '@testing-library/react'
```

Comment:
Not actually using React Testing Library here, so the original `act`was correct

Reply:
Reverted